docs/roadmap.md

```diff
@@ -79,8 +79,12 @@ metadata:
 spec:
   replicas: 5
   workloadRef:
+    apiVersion: apps/v1
     kind: Deployment
     name: guestbook
+  selector:
+    matchLabels:
+      app: guestbook
```

Comment:
I think we should remove this section since workloadRef is in v1.0

Also, selector is not necessary for Deployments -- if it is a Deployment, we get the selector from the deployment.

Reply:
> I think we should remove this section from roadmap since workloadRef is in v1.0
> 
> Also, selector is not necessary for Deployments -- if it is a Deployment, we get the selector from the deployment.

You are right. I was testing using v0.10.2 and saw the error of missing `selector` labels. The latest version does not requires them.

How about moving the section to v1.0?